### PR TITLE
Set specific version for DETAILED_BUILD_VERSION (used for M115)

### PR DESCRIPTION
**Pull request by Kevinrob**
### Requirements

* Filling out this template is required. Pull Requests without a clear description may be closed at the maintainers' discretion.

### Description

@MrBill123 ask in [an issue](https://github.com/knutwurst/Marlin-2-0-x-Anycubic-i3-MEGA-S/issues/296) if we can set specific version for `M115` command.
As I just flashed my printer and want to help to this repo, I made a quick PR for this.

### Benefits

Get the specific correct version

### Related Issues

Related to https://github.com/knutwurst/Marlin-2-0-x-Anycubic-i3-MEGA-S/issues/296

### CODE REVIEW

File: Marlin/src/inc/Version.h [hunk 1]
```diff
@@ -33,7 +33,7 @@
  * vendor name, download location, GitHub account, etc.
  */
 #ifndef DETAILED_BUILD_VERSION
-  #define DETAILED_BUILD_VERSION SHORT_BUILD_VERSION " (knutwurst, Github)" // PATCH: Knutwurst
+  #define DETAILED_BUILD_VERSION "2.0.9.2 (knutwurst, Github)" // PATCH: Knutwurst
```
Comment (Kevinrob) [inline Marlin/src/inc/Version.h:36]: @knutwurst Is there any reason to not change this?

Comment (MrBill1234) [reply Marlin/src/inc/Version.h:36]: Should it not be "Marlin 2.0.9.2 (knutwurst, Github)" 

Do you know if this will be merged into the base?

Comment (Kevinrob) [reply Marlin/src/inc/Version.h:36]: Currently, it's `2.0.x (knutwurst, Github)`.
I don't know 🙂

Comment (MrBill1234) [reply Marlin/src/inc/Version.h:36]: Current output is

Send: M115
Recv: FIRMWARE_NAME:Marlin 2.0.x (knutwurst, Github) (Feb 18 2022 13:26:03) SOURCE_CODE_URL:github.com/knutwurst/Marlin-2-0-x-Anycubic-i3-MEGA-S PROTOCOL_VERSION:1.0 MACHINE_TYPE:Knutwurst's Anycubic Printer EXTRUDER_COUNT:1 UUID:b6153152-86fb-4cd8-bda7-91c6a04c467d

it must pick up the ‘Marlin’ text from somewhere else?  Is it in SHORT_BUILD_VERSION?

**Comment by Kevinrob** [comment]
I see that it's fixed with https://github.com/knutwurst/Marlin-2-0-x-Anycubic-i3-MEGA-S/commit/41d069464e3f762c93502bec04245dc0866a19e8

**Pull request closed without merging**